pkg/apply/gvk.go

```diff
@@ -0,0 +1,96 @@
+// Copyright 2020 The Kubernetes Authors.
```

Comment:
This file seems to be copied from Kustomize. Aside from the list of resources to decide how we should sort, I'm not sure what we actually need.

Reply:
Will do.